vault/logical_system_paths.go

```diff
@@ -64,38 +84,208 @@ func (b *SystemBackend) configPaths() []*framework.Path {
+					Description: "Specifies a single master key share.",
+				},
+				"nonce": &framework.FieldSchema{
+					Type:        framework.TypeString,
+					Description: "Specifies the nonce of the attempt.",
+				},
+			},
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.UpdateOperation: &framework.PathOperation{
+					Summary:     "Enter a single master key share to progress the root generation attempt.",
+					Description: "If the threshold number of master key shares is reached, Vault will complete the root generation and issue the new token. Otherwise, this API must be called multiple times until that threshold is met. The attempt nonce must be provided with each call.",
+				},
+			},
+
 			HelpSynopsis:    strings.TrimSpace(sysHelp["generate-root"][0]),
 			HelpDescription: strings.TrimSpace(sysHelp["generate-root"][1]),
 		},
+		{
+			Pattern: "health$",
+
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ReadOperation: &framework.PathOperation{
+					Summary: "Returns the health status of Vault.",
+					Responses: map[int][]framework.Response{
+						200: {{Description: "initialized, unsealed, and active"}},
+						429: {{Description: "unsealed and standby"}},
+						472: {{Description: "data recovery mode replication secondary and active"}},
+						501: {{Description: "not initialized"}},
+						503: {{Description: "sealed"}},
+					},
+				},
+			},
+		},
 
 		{
-			Pattern:         "init$",
+			Pattern: "init$",
+			Fields: map[string]*framework.FieldSchema{
+				"pgp_keys": &framework.FieldSchema{
+					Type:        framework.TypeCommaStringSlice,
+					Description: "Specifies an array of PGP public keys used to encrypt the output unseal keys. Ordering is preserved. The keys must be base64-encoded from their original binary representation. The size of this array must be the same as secret_shares.",
+				},
+				"root_token_pgp_key": &framework.FieldSchema{
+					Type:        framework.TypeString,
+					Description: "Specifies a PGP public key used to encrypt the initial root token. The key must be base64-encoded from its original binary representation.",
+				},
+				"secret_shares": &framework.FieldSchema{
+					Type:        framework.TypeInt,
+					Description: "Specifies the number of shares to split the master key into.",
+				},
+				"secret_threshold": &framework.FieldSchema{
+					Type:        framework.TypeInt,
+					Description: "Specifies the number of shares required to reconstruct the master key. This must be less than or equal secret_shares. If using Vault HSM with auto-unsealing, this value must be the same as secret_shares.",
+				},
+				"stored_shares": &framework.FieldSchema{
+					Type:        framework.TypeInt,
+					Description: "Specifies the number of shares that should be encrypted by the HSM and stored for auto-unsealing. Currently must be the same as secret_shares.",
+				},
+				"recovery_pgp_keys": &framework.FieldSchema{
```

Comment:
Was there a reason to skip `recovery_shares` and `recovery_threshold`?

Reply:
Nope, just missed adding them.